stories/assets/js/navigation.js

```diff
@@ -125,18 +118,18 @@ export const navigationInitialize = (locale) => {
     $('.mobile-sub-menu').addClass('show');
   });
 
-  $('.mob-lang-switcher').click((ev) => {
+  $('.mob-lang-switcher').on('click', function(ev) {
     ev.preventDefault();
     $('.mob-sub-lang').addClass('show');
     $('.mobile-links').addClass('hide');
   });
 
-  $('.back-nav').click(() => {
+  $('.back-nav').on('click',() => {
```

Comment:
@Ashish-srij Can you add `function()` keyword here to maintain consistency with the code on line #121 and #132.